eng/performance/helix.proj

```diff
@@ -1,63 +1,145 @@
 <Project Sdk="Microsoft.DotNet.Helix.Sdk" DefaultTargets="Test">
 
-  <PropertyGroup Condition="'$(TargetsWindows)' == 'true'">
+  <PropertyGroup Condition="'$(TargetsWindows)' == 'true' AND '$(UseCoreRun)' == 'true'">
+    <PerformanceDirectory>%HELIX_WORKITEM_ROOT%\performance</PerformanceDirectory>
+    <HelixPreCommands>$(HelixPreCommands) &amp;&amp; robocopy /np /nfl /e %HELIX_CORRELATION_PAYLOAD%\performance $(PerformanceDirectory) /XD %HELIX_CORRELATION_PAYLOAD%\performance\.git</HelixPreCommands>
```

Comment:
It seems like we are setting the HelixPreCommands here and then resetting them a few lines down. Am I missing something here?

Reply:
This is with UseCoreRun == true set which is true for runs against a local runtime build, the reason for this is that in the runtime repo we set the precommands in the helix.proj, but in the performance repo we set them in the yaml file. So this was a way to merge the two .proj files. In the future I think consolidating it so that more of this is done inside the python scripts would be good.